core/pkg/eval/fractional_evaluation.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/twmb/murmur3"
 )
 
+const targetingKeyKey = "targetingKey"
```

Comment:
Nit - this fits to `json_evaluator.go` const section (with a comment to explain the purpose). This should be re-usable for other evaluators if neede.

Reply:
Good point. Thank you! Updated and comment added.